### PR TITLE
Create new contact UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by vysakh-poulose**
Fixes #5

- Added delete UI for contacts
- Added initial values and validations for contacts
- Added create UI for contacts

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane, Toastr } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import formInitialValues from "constants/formInitialValues";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Form.jsx:8]: Isn't this a literal constant? Please update the naming to reflect a literal constant.

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import formValidationSchemas from "constants/formValidationSchemas";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Form.jsx:9]: Not sure why this validation schema is put under another `constants` folder. It should be within the closest scope, which is `./constants.js`
```suggestion
import { VALIDATION_SCHEMA } from "constants";
```

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      logger.error(error);
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={formInitialValues.contactsForm}
+      onSubmit={handleSubmit}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Form.jsx:29]: Why do you need to set this? Keeping a button with `type=submit` would do, right?

**Review by yedhink — COMMENTED** [review]
@vysakh-poulose Few comments.

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx (hunk 1 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Contacts/Form.jsx:8]: Fixed in 9bf2fa8

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx (hunk 2 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Contacts/Form.jsx:9]: Fixed in 9bf2fa8

File: app/javascript/src/components/Dashboard/Contacts/Form.jsx (hunk 3 above)
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Contacts/Form.jsx:29]: Fixed in 6995dfb

**Pull request merged**